### PR TITLE
feat(lib): Export sentry hub

### PR DESCRIPTION
**Pull request by faris-imi**

### CODE REVIEW

File: lib/src/loader.ts [hunk 1]
```diff
@@ -62,6 +63,8 @@ export function hCaptchaLoader(params: ILoaderParams = { cleanup: true }): Promi
             reportapi: params.reportapi,
             endpoint: params.endpoint,
             host: params.host,
+            recaptchacompat: params.recaptchacompat,
+            hl: params.hl,
```
Comment (brdlyptrs) [inline lib/src/loader.ts:63]: Nice call, can you please update the props table in the README as well.

File: lib/src/sentry.ts [hunk 2]
```diff
@@ -31,14 +33,24 @@ export function initSentry(sentry: boolean) {
...
-    scope.setTag('source', '@hCaptcha/loader');
-  });
+export function getSentry(): SentryHub | null {
+  return getSentryHubWrapper(hub);
+}
 
-  return hub;
+function setScopeTag(source: string = '@hCaptcha/loader') {
+  hub?.configureScope(function(scope) {
+    scope.setTag('source', source);
```
Comment (brdlyptrs) [inline lib/src/sentry.ts]: Shouldn't this be more generic incase you want to set a `key | value`? In this case having two parameters to pass in.

Comment (faris-imi) [reply lib/src/sentry.ts]: yes, changed

File: lib/src/sentry.ts [hunk 3]
```diff
@@ -31,14 +33,24 @@ export function initSentry(sentry: boolean) {
...
 }
 
-export function getSentry() {
-  return hub;
+function getSentryHubWrapper(sentryHub): SentryHub | null {
+  return {
+    captureMessage: (message) => sentryHub.captureMessage(message),
+    captureException: (params) => sentryHub.captureException(params),
+    addBreadcrumb: (params) => sentryHub.addBreadcrumb(params),
+    setTag: (source) => setScopeTag(source),
```
Comment (brdlyptrs) [inline lib/src/sentry.ts]: Believe you need to update this to now include the `key, value` as well.

Comment (faris-imi) [reply lib/src/sentry.ts]: wrapped these using `withScope` here https://github.com/hCaptcha/hcaptcha-loader/pull/8/commits/81abaf102445bdc2c358bed3024a94d0e2b0c7e3, which means calling this and passing tag will guarantee correct source is set

**Review by brdlyptrs — APPROVED** [review]

**Review by brdlyptrs — DISMISSED** [review]

**Review by brdlyptrs — APPROVED** [review]

**Pull request merged**